### PR TITLE
Update Overview.md

### PR DESCRIPTION
**Pull request by dhil**
This is a quick pass over the overview document. I have added type annotations and the tag filter list on `resume_throw`.

### CODE REVIEW

File: proposals/continuations/Overview.md [hunk 1]
```diff
@@ -19,20 +19,20 @@ Based on [typed reference proposal](https://github.com/WebAssembly/function-refe
...
-  - `cont.bind $ct : [t3* (ref null? $ct')] -> [(ref $ct)]`
+* `cont.bind <typeidx> <typeidx>` binds a continuation to (partial) arguments
+  - `cont.bind $ct $ct' : [t3* (ref null? $ct)] -> [(ref $ct')]`
     - iff `$ct = cont $ft`
-    - and `$ft = [t1*] -> [t2*]`
+    - and `$ft = [t3* t1*] -> [t2*]`
     - and `$ct' = cont $ft'`
-    - and `$ft' = [t3* t1'*] -> [t2'*]`
+    - and `$ft' = [t1'*] -> [t2'*]`
     - and `[t1'*] -> [t2'*] <: [t1*] -> [t2*]`
```
Comment (rossberg) [inline proposals/continuations/Overview.md]: If you switch ct and ct', then this needs changing sides as well.

File: proposals/continuations/Overview.md [hunk 2]
```diff
@@ -41,11 +41,16 @@ Based on [typed reference proposal](https://github.com/WebAssembly/function-refe
     - and `($ct' = cont $ft')*`
     - and `([te2*] -> [t2*] <: $ft')*`
```
Comment (rossberg) [inline proposals/continuations/Overview.md]: To address Frank's concern:
```suggestion
    - and `$ft' = [t1'*] -> [t2'*]`
    - and `([te2*] -> [t2*] <: [t1'*] -> [t2'*])*`
```
(Subtyping on raw function types is defined structurally, so this works.)

File: proposals/continuations/Overview.md [hunk 3]
```diff
@@ -41,11 +41,16 @@ Based on [typed reference proposal](https://github.com/WebAssembly/function-refe
...
+* `resume_throw <typeidx> <tagidx> (tag <tagidx> <labelidx>)` aborts a continuation
+  - `resume_throw $ct $e (tag $t $l): [te* (ref null? $ct)] -> [t2*]`
+    - iff `(tag $e : [te*] -> [])`
     - and `$ct = cont $ft`
     - and `$ft = [t1*] -> [t2*]`
+    - and `(tag $t : [te1*] -> [te2*])*`
+    - and `(label $l : [te1'* (ref null? $ct')])*`
+    - and `([te1*] <: [te1'*])*`
+    - and `($ct' = cont $ft')*`
+    - and `([te2*] -> [t2*] <: $ft')*`
```
Comment (rossberg) [inline proposals/continuations/Overview.md]: Same here.

File: proposals/continuations/Overview.md [hunk 4]
```diff
@@ -111,36 +116,37 @@ H^ea ::=
...
-* `S; F; (ref.cont ca) (cont.bind $ct)  -->  S'; F; trap`
+* `S; F; (ref.cont ca) (cont.bind $ct $ct')  -->  S'; F; trap`
   - iff `S.conts[ca] = epsilon`
 
-* `S; F; v^n (ref.cont ca) (cont.bind $ct)  -->  S'; F; (ref.const |S.conts|)`
+* `S; F; v^n (ref.cont ca) (cont.bind $ct $ct')  -->  S'; F; (ref.const |S.conts|)`
   - iff `S.conts[ca] = (E' : n')`
-  - and `$ct = cont $ft`
-  - and `$ft = [t1'*] -> [t2'*]`
+  - and `$ct' = cont $ft`
```
Comment (rossberg) [inline proposals/continuations/Overview.md]:
```suggestion
  - and `$ct' = cont $ft'`
```

**Review by rossberg — APPROVED** [review]
LGTM modulo comments.

**Comment by dhil** [comment]
Thanks @rossberg! I have addressed your comments in the latest commit. @frank-emrich have you had a look yet?

**Pull request merged**